utils/run_test.sh

```diff
@@ -130,8 +130,30 @@ if [ -d "/mnt/daos" ]; then
     run_test "${SL_BUILD_DIR}/src/gurt/tests/test_gurt_telem_producer"
 
     COMP="UTEST_vos"
-    run_test "${SL_PREFIX}/bin/vos_tests" -A 500
-    run_test "${SL_PREFIX}/bin/vos_tests" -n -A 500
+    # Setup hugepages
+    NR_HPS=$(sysctl vm.nr_hugepages -n)
+    if [ "${NR_HPS}" -lt 1024 ]; then
+	sudo sysctl vm.nr_hugepages=1024
+    fi
+    NR_HPS=$(sysctl vm.nr_hugepages -n)
+    echo "nr_hugepages="${NR_HPS}""
```

Comment:
(lint) The surrounding quotes actually unquote this. Remove or escape them. [SC2027]
(lint) Double quote to prevent globbing and word splitting. [SC2086]